website/docs/components/table/partials/specifications/anatomy.md

```diff
@@ -18,4 +18,22 @@
 |--------------|----------|
 | Cell content | Required |
 | Icon         | Optional |
+| Container    | Required |
+
+### Table header select 
```

Comment:
I'm not sure if having multiple sections here makes sense. Open to other opinions, but I might consider replacing "Table headers" and "Table cells" with the new graphics you created and combining them somehow. Maybe something like this:

**Table header image (with the selection component)**

| Element | Usage |
|---------|--------|
| Label | Required |
| Sort | Optional; indeterminate, ascending, descending |
| Container | Optional |
| Selection | Optional |

_Also note that this is going to change with the sort/tooltip updates, just doing this to illustrate. Though now that I'm thinking through it we should maybe consider a convergence point of the doc updates_

Reply:
@andgen404 has this been addressed? if so, please resolve the conversation